app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -78,22 +78,18 @@ export const Notifications = ({ setHeaderTitleNode }) => {
     dispatch(updateNotificationStateAction(isEnabled));
   };
 
-  const confirmAdd = (withoutAttributes) => (newNotification) => {
-    const notificationData = { ...newNotification };
-    if (withoutAttributes) {
-      notificationData.attributes = [];
-    }
-    const notification = convertNotificationCaseForSubmission(notificationData);
+  const confirmAdd = (newNotification) => {
+    const notification = convertNotificationCaseForSubmission(newNotification);
     dispatch(addProjectNotificationAction(notification));
   };
 
-  const confirmEdit = (withoutAttributes) => (notification) => {
+  const confirmEdit = (notification) => {
     dispatch(
       updateProjectNotificationAction(
         convertNotificationCaseForSubmission({
           ...notification,
           name: notification.ruleName,
-          attributes: withoutAttributes ? [] : notification.attributes,
+          attributes: notification.attributes,
```

Comment:
Do you really need it here? You spread notification and right after that rewrite attribute from notification